diff --git a/node_modules/@morbidick/lit-element-notify/notify.d.ts b/node_modules/@morbidick/lit-element-notify/notify.d.ts
index 3524496..ff12e56 100644
--- a/node_modules/@morbidick/lit-element-notify/notify.d.ts
+++ b/node_modules/@morbidick/lit-element-notify/notify.d.ts
@@ -1,14 +1,12 @@
-import { PropertyDeclaration, LitElement, UpdatingElement } from 'lit-element';
+import { LitElement, UpdatingElement } from 'lit-element';
 
 type Constructor<T = LitElement> = new (...args: any[]) => T;
 
-interface AugmentedPropertyDeclaration extends PropertyDeclaration {
-  /** When true will notify. Pass a string to define the event name to fire. */
-  notify: string|Boolean
+declare module 'lit-element/lib/updating-element' {
+  export interface PropertyDeclaration {
+    /** When true will notify. Pass a string to define the event name to fire. */
+    notify?: string|Boolean
+  }
 }
 
-declare class NotifyingElement {
-  static createProperty(name: string, options: AugmentedPropertyDeclaration): void
-}
-
-export function LitNotify<T extends UpdatingElement>(baseElement: Constructor<T>): T & NotifyingElement
+export function LitNotify<T extends Constructor<UpdatingElement>>(baseElement: T): T
