diff --git a/node_modules/@open-wc/lit-helpers/src/read-only-properties-mixin.d.ts b/node_modules/@open-wc/lit-helpers/src/read-only-properties-mixin.d.ts
index b6aba38..d60dd80 100644
--- a/node_modules/@open-wc/lit-helpers/src/read-only-properties-mixin.d.ts
+++ b/node_modules/@open-wc/lit-helpers/src/read-only-properties-mixin.d.ts
@@ -1,8 +1,4 @@
-// when we update to typescript 3.8
-// import type { LitElement } from "lit-element";
-import { LitElement } from "lit-element";
-
-declare type Constructor<T = {}> = new (...args: any[]) => T;
+import type { LitElement, Constructor } from "lit-element";
 
 declare module 'lit-element/lib/updating-element' {
   interface PropertyDeclaration {
@@ -10,18 +6,18 @@ declare module 'lit-element/lib/updating-element' {
   }
 }
 
-export interface ReadOnlyClass {
+/**
+ * Enables the optional readOnly property on property declarations,
+ * and adds `setReadOnlyProperties(properties)` to elements
+ */
+export declare function ReadOnlyPropertiesMixin<T extends Constructor<LitElement>>(superclass: T): T & Constructor<{
   /**
    * Set read-only properties
+   * @protected
    */
   setReadOnlyProperties(props: {
     [s: string]: unknown;
   }): Promise<void>;
-}
-
-/**
- * Enables the nofity option for properties to fire change notification events
- */
-export declare function ReadOnlyPropertiesMixin<T extends Constructor<LitElement>>(superclass: T): T & Constructor<ReadOnlyClass>;
+}>;
 
 export {};
